onnxruntime/core/providers/webgpu/tensor/pad.wgsl.template

```diff
@@ -0,0 +1,74 @@
+#define PAD_MODE_CONSTANT 0
```

Comment:
will #include be supported ?

Reply:
please see https://github.com/fs-eire/wgsl-template?tab=readme-ov-file#template-syntax for explanation of supported syntax